control-plane/subcommand/inject-connect/command.go

```diff
@@ -738,12 +750,21 @@ func (c *Command) validateFlags() error {
 	if c.flagConsulK8sImage == "" {
 		return errors.New("-consul-k8s-image must be set")
 	}
+	if c.flagConsulK8sImageWindows == "" {
+		return errors.New("-consul-k8s-image-windows must be set")
+	}
```

Comment:
I don't think we should require these images to be set. For linux only deployments, I'd expect these to be empty.